MIGRATION.md

```diff
@@ -11,6 +11,7 @@
   - Related: `EmulatorBuilder` uses a single function to provide a `Qemu` initializer: `EmulatorBuilder::qemu_parameters`. For now, it can be either a `Vec<String>` or a `QemuConfig` instance.
   - Related: Qemu's `AsanModule` does not need any special call to `Qemu` init methods anymore. It is now possible to simply initialize `AsanModule` (or `AsanGuestModule`) with a reference to the environment as parameter.
   - `CustomBufHandlers` has been deleted. Please use `EventManagerHooksTuple` from now on.
+  - Three traits, `State`, `UsesState`, `UsesInput` has been deprecated.
```

Comment:
I think `State` should stay, just as a collection of traits you need to implement if you want to create your own state

Reply:
ok i'll keep the definition. but remove all usage of these

Reply:
And make StdState implement it, mainly for documentation